### PR TITLE
#68: described new version and release policy

### PR DESCRIPTION
**Pull request by alfonso-noriega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,26 @@ The search lib is modular and currently implements the following layers:
 
 ![Search Lib Architecture](./docs/images/layer_cake.png)
 
+## Versioning & release policy
+
+Each Vind released is labeled in the repository with a tag fitting the following schema: _vind-va.b.c_ where 
+**vind-v** just points out that this belongs to a Vind version and **a.b.c** represents the release number, also used 
+for the artifact version. Prior to **vind-v1.3.0** there was not a clear policy on versioning therefor, there was also not 
+an explanation about it.
```
Comment (tkurz) [inline README.md]: remove  'therefor ... .'

File: README.md [hunk 2]
```diff
@@ -28,6 +28,26 @@ The search lib is modular and currently implements the following layers:
 
 ![Search Lib Architecture](./docs/images/layer_cake.png)
 
+## Versioning & release policy
+
+Each Vind released is labeled in the repository with a tag fitting the following schema: _vind-va.b.c_ where 
+**vind-v** just points out that this belongs to a Vind version and **a.b.c** represents the release number, also used 
+for the artifact version. Prior to **vind-v1.3.0** there was not a clear policy on versioning therefor, there was also not 
+an explanation about it. 
+From __1.3.0__ on the releases will strictly stick to the following diagram:
```
Comment (tkurz) [inline README.md]: remove 'on'

File: README.md [hunk 3]
```diff
@@ -28,6 +28,26 @@ The search lib is modular and currently implements the following layers:
...
+
+![Versioning policy](./docs/images/versioning_policy.png)
+
+Regarding the release process, prior to every non _hot fix_ release, a **release candidate** would be published to be tested 
+on an staging environment as closest as possible to production. These releases will apply the previous described name schema
```
Comment (tkurz) [inline README.md]: closest -> close

File: README.md [hunk 4]
```diff
@@ -28,6 +28,26 @@ The search lib is modular and currently implements the following layers:
...
 
+## Versioning & release policy
+
+Each Vind released is labeled in the repository with a tag fitting the following schema: _vind-va.b.c_ where 
+**vind-v** just points out that this belongs to a Vind version and **a.b.c** represents the release number, also used 
+for the artifact version. Prior to **vind-v1.3.0** there was not a clear policy on versioning therefor, there was also not 
+an explanation about it. 
+From __1.3.0__ on the releases will strictly stick to the following diagram:
+
+![Versioning policy](./docs/images/versioning_policy.png)
```
Comment (tkurz) [inline README.md]: add on the first number add something like "this includes version changes where a complete reindex is necessary to keep the system working"

File: README.md [hunk 5]
```diff
@@ -28,6 +28,26 @@ The search lib is modular and currently implements the following layers:
 
 ![Search Lib Architecture](./docs/images/layer_cake.png)
 
+## Versioning & release policy
+
+Each Vind released is labeled in the repository with a tag fitting the following schema: _vind-va.b.c_ where
```
Comment (mhaslauer) [inline README.md]: released -> release

File: README.md [hunk 6]
```diff
@@ -28,6 +28,26 @@ The search lib is modular and currently implements the following layers:
 
 ![Search Lib Architecture](./docs/images/layer_cake.png)
 
+## Versioning & release policy
+
+Each Vind released is labeled in the repository with a tag fitting the following schema: _vind-va.b.c_ where 
+**vind-v** just points out that this belongs to a Vind version and **a.b.c** represents the release number, also used 
+for the artifact version. Prior to **vind-v1.3.0** there was not a clear policy on versioning.
+
+From __1.3.0__ the releases will strictly stick to the following diagram:
```
Comment (mhaslauer) [inline README.md]: I would remove the *the* and add *on*, so -> From __1.3.0__ on releases will strictly stick to the following diagram:

**Pull request merged**